src/dataflow/render/reduce.rs

```diff
@@ -215,18 +274,23 @@ where
         // The same code should work on data that can not be hierarchically reduced.
         partial.reduce_abelian::<_, OrdValSpine<_, _, _, _>>("ReduceInaccumulable", {
             move |key, source, target| {
-                // We respect the multiplicity here (unlike in hierarchical aggregation)
-                // because we don't know that the aggregation method is not sensitive
-                // to the number of records.
-                let iter = source.iter().flat_map(|(v, w)| {
-                    std::iter::repeat(v.iter().next().unwrap()).take(*w as usize)
-                });
-                let mut packer = RowPacker::new();
-                if prepend_key {
-                    packer.extend(key.iter());
+                if source.iter().any(|(_val, cnt)| cnt <= &0) {
```

Comment:
Why `<=` instead of `<` ?

Reply:
I forget! We shouldn't be seeing zero either, so I'm happy to yell in that case too. But the yelling is not very correct is it. Will fix.

Reply:
Yep, should say "nonpositive" instead of "negative"